3p/iframe-messaging-client.js

```diff
@@ -33,7 +33,7 @@ export class IframeMessagingClient {
     this.win_ = win;
     /** @private {?string} */
     this.rtvVersion_ = getMode().rtvVersion || null;
-    /** @private {!Window} */
+    /** @private {!(Window|Element)} */
```

Comment:
Tried this originally, but unfortunately there is not.
`
3p/iframe-messaging-client.js:131: ERROR - Bad type annotation. Unknown type Frame
   * @param {!(Window|Frame)} win The window to communicate with. This may
`